<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="del_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="words_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Посмотреть слова</string>
   </property>
  </widget>
  <widget class="QPushButton" name="colls_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Посмотреть сочетания</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
